podman_compose.py

```diff
@@ -857,16 +857,15 @@ def run(self, podman_args, cmd='', cmd_args=None, wait=True, sleep=1, obj=None):
             time.sleep(sleep)
         return p
 
-    def volume_inspect_all(self):
-        output = self.output([], "volume", ["inspect", "--all"]).decode('utf-8')
-        return json.loads(output)
-    
-    def volume_inspect_proj(self, proj=None):
+    def volume_ls(self, proj=None):
         if not proj:
             proj = self.compose.project_name
-        volumes = [(vol.get("Labels", {}), vol) for vol in self.volume_inspect_all()]
-        volumes = [(labels.get("io.podman.compose.project", None), vol) for labels, vol in volumes]
-        return [vol for vol_proj, vol in volumes if vol_proj==proj]
+        output = self.output([], "volume", [
+            "ls", "--noheading", "--filter", "label=io.podman.compose.project=cachito",
```

Comment:
```
            "ls", "--noheading", "--filter", f"label=io.podman.compose.project={proj}",
```

Reply:
Ugh.. my bad. Thanks for catching this. Fixed.